homeassistant/components/fritz/button.py

```diff
@@ -30,7 +30,7 @@
 class FritzButtonDescription(ButtonEntityDescription):
     """Class to describe a Button entity."""
 
-    press_action: Callable
+    press_action: Callable[[AvmWrapper], Any]
```

Comment:
i know it is not really related, but found this while checking if the button entities somehow rely on the services or directly call the corresponding functions